package-lock.json

```diff
@@ -11761,12 +11758,6 @@
         "postcss": "^7.0.14"
       }
     },
-    "icu4c-data": {
```

Comment:
should these be excluded from the change?

Reply:
This is going to get another update again when we update the git commit ID again, so I wouldn't sweat it yet.